src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub struct Check {
     release_type: Option<ReleaseType>,
     current_feature_config: rustdoc_gen::FeatureConfig,
     baseline_feature_config: rustdoc_gen::FeatureConfig,
+    target: Option<String>,
```

Comment:
Probably a good idea to add a field-level doc comment here, describing what the field means and what the value `None` means.